### PR TITLE
docs(radio): Add README.

### PR DESCRIPTION
**Pull request by rfru**
@jelbourn

### CODE REVIEW

File: src/components/radio/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## `<md-radio-button>`
+### Properties
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `id` | `String` | The unique ID of this radio button. |
+| `name` | `String` | Group name, defaults to parent radio group if present. |
+| `value` | `any` | The value of this radio button. |
+| `checked` | `Boolean` | Whether the radio is checked. |
+| `disabled` | `Boolean` | Whether the radio is disabled. |
```
Comment (jelbourn) [inline src/components/radio/README.md]: Should be `boolean` and `string` lower-case.

Comment (rfru) [inline src/components/radio/README.md]: Fixed.

File: src/components/radio/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# MdRadio
+Radio buttons allow the user to select one option from a set. Use radio buttons for exclusive selection if you think that the user needs to see all available options side-by-side.
+
+![Preview](https://material-design.storage.googleapis.com/publish/material_v_4/material_ext_publish/0Bx4BSt6jniD7Z1NaaXh2ZkpDRkE/components_switches_radio1.png)
```
Comment (jelbourn) [inline src/components/radio/README.md]: I updated a screenshot for you:
https://material.angularjs.org/material2_assets/radio/radios.png

Comment (rfru) [inline src/components/radio/README.md]: Cool, thanks! Updated.

**Pull request closed without merging**